op-batcher/batcher/channel_builder.go

```diff
@@ -127,12 +134,14 @@ type channelBuilder struct {
 
 // newChannelBuilder creates a new channel builder or returns an error if the
 // channel out could not be created.
-func newChannelBuilder(cfg ChannelConfig) (*channelBuilder, error) {
+func newChannelBuilder(cfg ChannelConfig, rcfg *rollup.Config) (*channelBuilder, error) {
 	c, err := cfg.CompressorConfig.NewCompressor()
 	if err != nil {
 		return nil, err
 	}
-	co, err := derive.NewChannelOut(c)
+	// Creates a spanBatchBuilder contains information requires to build SpanBatch
+	spanBatchBuilder := derive.NewSpanBatchBuilder(cfg.ParentRef.L1Origin.Number, rcfg.Genesis.L2Time, rcfg.L2ChainID)
```

Comment:
We do not have to initialize `spanBatchBuilder`. if singular batch, lets set `spanBatchBuilder` as nil.